alchemist-graphql/build.gradle.kts

```diff
@@ -124,6 +141,15 @@ tasks.named("run", JavaExec::class).configure {
     )
 }
 
+/**
+ * Configure GraphQL Schema download task to download the schema from the server
+ * at the default endpoint, locating the updated schema under the correct directory.
+ */
+tasks.withType<com.expediagroup.graphql.plugin.gradle.tasks.GraphQLDownloadSDLTask>().configureEach {
+    outputFile.set(File("src/commonMain/resources/graphql/schema.graphqls"))
+    endpoint = "http://localhost:8081/sdl"
```

Comment:
what is sdl for?

Reply:
it stands for Schema Definition Language, and `GraphQLDownloadSDL` task downloads the schema at the provided endpoint.